File: BOJ/CPP/단계별로_풀어보기/03_반복문/BOJ_2742.cpp
/*
	BOJ 2742 : 기찍 N

	author : kdgyun
	link https://st-lab.tistory.com
	link https://github.com/kdgyun
	Created by kdgyun
	Copyright © 2021 kdgyun. All rights reserved.
*/

#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {

	ios_base::sync_with_stdio(false);

	int N;
	cin >> N;

	for (int i = N; i > 0; i--) {
		cout << i << "\n";
	}
	return 0;
}

File: BOJ/Java/단계별로_풀어보기/B_03_반복문/BOJ_2742.java
package 단계별로_풀어보기.B_03_반복문;

/**
 * BOJ 2742 : 기찍 N
 * 
 * @author kdgyun
 * {@link https://st-lab.tistory.com}
 * {@link https://github.com/kdgyun}
 *
 */

import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class BOJ_2742 {

	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		int N = Integer.parseInt(br.readLine());
		br.close();
        
		StringBuilder sb = new StringBuilder();
        
		while(N > 0) {
			sb.append(N--).append('\n');

		}
		System.out.println(sb);
	}
}
